src/components/CheckboxField/README.md

```diff
@@ -186,6 +186,76 @@ React.createElement(() => {
 });
 ```
 
+### Required State
```

Comment:
> But it is true that one more sentence could be helpful. I am missing the detailed explanation why would user might need this and why it is correct approach.

I rewrote the docs and reworked the example. What do you guys think now?

<img width="954" alt="obrazek" src="https://github.com/user-attachments/assets/f3937405-6078-4094-8607-8fb42aa75197">

Reply:
Suggestions:

- emphasize it's just a visual thing
- when a checkbox can be true or false, not only true
- consider moving it aside as an edge case

Reply:
I like it.